### PR TITLE
[docs] Smart driver with verify-full and YBM

### PR DESCRIPTION
**Pull request by ddhodge**
Notes on using smart drivers with YBM clusters and verify-full SSL mode

@netlify /preview/drivers-orms/smart-drivers/#using-smart-drivers-with-yugabytedb-managed

### CODE REVIEW

File: docs/content/preview/reference/drivers/go/yb-pgx-reference.md [hunk 1]
```diff
@@ -166,7 +145,9 @@ By default, the YugabyteDB PGX driver automatically prepares and caches statemen
 #### Query data
 
 To query data from YugabyteDB tables, execute the `SELECT` statement using the function `conn.Query()`.
-Query results are returned in `pgx.Rows` which can be iterated using `pgx.Rows.next()` method.
+
+Query results are returned in `pgx.Rows`, which can be iterated using the `pgx.Rows.next()` method.
+
 Then read the data using `pgx.rows.Scan()`.
 
 The SELECT DML statement:
```
Comment (lizayugabyte) [inline docs/content/preview/reference/drivers/go/yb-pgx-reference.md:153]: `SELECT`

File: docs/content/preview/reference/drivers/python/yugabyte-psycopg2-reference.md [hunk 2]
```diff
@@ -128,16 +125,18 @@ This tutorial shows how to use the Yugabyte Psycopg2 driver with YugabyteDB. It
 Next, you use Python shell terminal to demonstrate the driver's load balancing features by running a few python scripts.
 
 {{< note title="Note">}}
-The driver requires YugabyteDB version 2.7.2.0 or higher
+The driver requires YugabyteDB version 2.7.2.0 or higher.
```
Comment (lizayugabyte) [inline docs/content/preview/reference/drivers/python/yugabyte-psycopg2-reference.md:128]: or later.

File: docs/content/preview/reference/drivers/python/yugabyte-psycopg2-reference.md [hunk 3]
```diff
@@ -128,16 +125,18 @@ This tutorial shows how to use the Yugabyte Psycopg2 driver with YugabyteDB. It
...
 {{< /note>}}
 
-### Install YugabyteDB and create a local cluster
+### Create a local cluster
 
 Create a universe with a 3-node RF-3 cluster with some fictitious geo-locations assigned. The placement values used are just tokens and have nothing to do with actual AWS cloud regions and zones.
 
 ```sh
 $ cd <path-to-yugabytedb-installation>
```
Comment (lizayugabyte) [inline docs/content/preview/reference/drivers/python/yugabyte-psycopg2-reference.md:136]: delete prompt

**Review by lizayugabyte — APPROVED** [review]
minor comments

**Comment by ddhodge** [comment]
@ashetkar Can you take a look, or are we waiting on some things?

File: docs/content/preview/drivers-orms/java/yugabyte-jdbc.md [hunk 4]
```diff
@@ -141,6 +148,8 @@ jdbc:yugabytedb://hostname:port/database?user=yugabyte&password=yugabyte&load-ba
 
 If you created a cluster on [YugabyteDB Managed](https://www.yugabyte.com/managed/), use the cluster credentials and [download the SSL Root certificate](../../../yugabyte-cloud/cloud-connect/connect-applications/).
 
+To use load balancing and SSL mode verify-full with a cluster in YugabyteDB Managed, you need to provide the additional `sslhostnameverifier` parameter, set to `com.yugabyte.ysql.YBManagedHostnameVerifier`.
```
Comment (ashetkar) [inline docs/content/preview/drivers-orms/java/yugabyte-jdbc.md:151]: This property/param is available from version 42.3.5-yb-2 onwards.

**Review by ashetkar — APPROVED** [review]

**Pull request merged**